python/apps/trolli/src/board_list.py

```diff
@@ -0,0 +1,236 @@
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from board import Board
+import itertools
+from flet import (
+    UserControl,
+    Draggable,
+    DragTarget,
+    Column,
+    Row,
+    Text,
+    Icon,
+    PopupMenuButton,
+    PopupMenuItem,
+    Container,
+    TextButton,
+    TextField,
+    icons,
+    border_radius,
+    alignment,
+    border,
+    colors,
+    padding,
+)
+from item import Item
+from memory_store import store
+from data_store import DataStore
+
+
+class BoardList(UserControl):
+    id_counter = itertools.count()
+
+    def __init__(self, board: "Board", title: str, color: str = ""):
+        super().__init__()
+        self.board_list_id = next(BoardList.id_counter)
+        self.store: DataStore = store
+        self.board = board
+        self.title = title
+        self.color = color
+        self.items = Column([], tight=True, spacing=4)
+        self.items.controls = self.store.get_items(self.board_list_id)
+
+    def build(self):
+
+        self.new_item_field = TextField(
+            label="new card name", height=50, bgcolor=colors.WHITE)
```

Comment:
It will be great if this TextField could have an `on_submit=self.add_item_handler`
Many don't like using the mouse alot. :)

Reply:
Right, I had meant to add that actually. Thanks.